### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by IViol**

### CODE REVIEW

File: src/store/modules/wallets.js [hunk 1]
```diff
@@ -237,104 +70,26 @@ export function simplifyWallet(walletId: WalletId, isSimplified: boolean) {
...
   ExtractReturn<typeof setWalletsItems> |
   ExtractReturn<typeof setActiveWallet> |
-  ExtractReturn<typeof setIsLoading> |
-  ExtractReturn<typeof setInvalidField> |
   ExtractReturn<typeof privateKeyError> |
   ExtractReturn<typeof privateKeySuccess> |
   ExtractReturn<typeof privateKeyRequest> |
-  ExtractReturn<typeof simplifyWallet> |
-  ExtractReturn<typeof clean>
+  ExtractReturn<typeof simplifyWallet>
```
Comment (Creepypoke) [inline src/store/modules/wallets.js]: implementation of `simplifyWallet` already exist?

File: src/store/sagas/upgrade.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 import {
   put,
   select,
   takeEvery,
 } from 'redux-saga/effects'
 
-import walletsWorker from 'workers/wallets'
 import { router5BackOrFallbackFunctionCreator } from 'utils/browser'
-import { selectWallets } from 'store/selectors/wallets'
+// import { selectWallets } from 'store/selectors/wallets'
```
Comment (Creepypoke) [inline src/store/sagas/upgrade.js]: Why just don't remove the line?

File: src/store/sagas/upgrade.js [hunk 3]
```diff
@@ -25,8 +24,9 @@ function* upgradeSuccess(action: ExtractReturn<typeof upgrade.upgradeSuccess>):
 }
 
 function* submitMnemonicRequest(
-  action: ExtractReturn<typeof upgrade.submitMnemonicRequest>,
+// action: ExtractReturn<typeof upgrade.submitMnemonicRequest>,
```
Comment (Creepypoke) [inline src/store/sagas/upgrade.js]: same, about removing lines

Comment (IViol) [reply src/store/sagas/upgrade.js]: lines will be uncommented in few days, once we are ready to update these screens

File: src/store/modules/wallets.js (hunk 1 above)
Comment (IViol) [reply src/store/modules/wallets.js]: yes

**Pull request merged**